code/modules/food/food/condiment.dm

```diff
@@ -204,15 +221,25 @@
 	reagents.add_reagent("sodiumchloride", 20)
 
 /obj/item/weapon/reagent_containers/food/condiment/small/peppermill
-	name = "pepper mill"
+	name = "pepper shaker"
```

Comment:
As briefly discussed on Discord I think this object should be repathed to have the name and item type align.